step.sh

```diff
@@ -30,7 +31,7 @@ printf "Obtaining a Token - Done\n"
 
 printf "\nObtaining the File Upload URL\n"
 curl --silent -X GET \
-  'https://connect-api.cloud.huawei.com/api/publish/v2/upload-url?appid='"${huawei_app_id}"'&suffix=apk' \
+  'https://connect-api.cloud.huawei.com/api/publish/v2/upload-url?appid='"${huawei_app_id}"'&suffix='"${FILE_EXT}" \
```

Comment:
@pawelcala 

The last apostrophe should be after FILE_EXT var like below, right?

`'https://connect-api.cloud.huawei.com/api/publish/v2/upload-url?appid='"${huawei_app_id}"'&suffix="${FILE_EXT}"'`

Reply:
@klamborowski yes, I currently have this running in a bitrise flow